### PR TITLE
add kube conformance variant

### PR DESCRIPTION
**Pull request by deads2k**

### CODE REVIEW

File: pkg/testgridanalysis/testidentification/kube_variants.go [hunk 1]
```diff
@@ -9,14 +9,16 @@ import (
 
 var (
 	// variant regexes
-	kubeKindRegex    = regexp.MustCompile(`kind-`)
-	kubeKubeadmRegex = regexp.MustCompile(`kubeadm-`)
-	kubeSerialRegex  = regexp.MustCompile(`-serial`)
-	kubeWindowsRegex = regexp.MustCompile(`windows-`)
-	kubeUpgradeRegex = regexp.MustCompile(`upgrade-`)
-	kubeE2eRegex     = regexp.MustCompile(`-parallel`)
+	kubeConformanceRegex = regexp.MustCompile(`onformance-`)
```
Comment (deads2k) [inline pkg/testgridanalysis/testidentification/kube_variants.go:12]: they have cap C and lower c and I dont' speak regex, but this is uncommon.

Comment (bparees) [reply pkg/testgridanalysis/testidentification/kube_variants.go:12]: `[Cc]onformance-` should do it.

or you might at least want a comment on this so some "helpful" person doesn't add a C in the future.

Comment (deads2k) [reply pkg/testgridanalysis/testidentification/kube_variants.go]: Updated.

**Pull request merged**